src/Microsoft.ComponentDetection.Detectors/rust/Contracts/CargoLock.cs

```diff
@@ -1,13 +1,15 @@
-﻿using System.Diagnostics.CodeAnalysis;
-using Nett;
+using System.Diagnostics.CodeAnalysis;
 
 namespace Microsoft.ComponentDetection.Detectors.Rust.Contracts
 {
+    using System.Collections.Generic;
+    using Tomlyn.Model;
+
     // Represents Cargo.Lock file structure.
     public class CargoLock
     {
         [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1300:ElementMustBeginWithUpperCaseLetter", Justification = "Deserialization contract. Casing cannot be overwritten.")]
-        public CargoPackage[] package { get; set; }
+        public List<CargoPackage> package { get; set; }
```

Comment:
Same here:

```suggestion
        [DataMember(Name = "package")]
        public IEnumerable<CargoPackage> Package { get; set; }
```

Also remove `[SuppressMessage]` from all properties, and add `[DataContract]` attribute to class please.